src/main/java/org/fundacionjala/coding/oscar/Kata1.java

```diff
@@ -0,0 +1,23 @@
+package org.fundacionjala.coding.oscar;
+
+import java.util.Arrays;
+
+/**
+ * Created by Oz64 on 17/08/2017.
+ */
+public class Kata1 {
+    /**
+     *  This function return the mayor and minus numbers in a string.
+     * @param stringNumbers
+     * is a string of numbers
+     * @return
+     * a string with the mayor and minor.
+     */
+
+    public String highAndLow(String stringNumbers) {
+        String[] arrayNumbers = stringNumbers.split(" ");
+        Arrays.sort(arrayNumbers);
+        String result = arrayNumbers[arrayNumbers.length - 1] + " " + arrayNumbers[0];
```

Comment:
Don't concatenate Strings using + 
please use a String.format instead

and you can return directly the String value
you don't need to store result variable because that variable will not be used.